.travis.yml

```diff
@@ -68,6 +68,13 @@ jobs:
                   tags: true
                   all_branches: true
 
+      after_deploy:
+          -  export TRAVIS_BRANCH
```

Comment:
use IMAGE_TAG var, that will have tags and branches
var should be already exported to env in line 43